### DISCUSSION TITLE
How to configure the logs in the console so that every SQL statement isn't displayed

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#1146] [Q&A] [answered]
I'm finding that when I'm primarily focused on Svelte work that the SQL console logging is interfering with my ability to see the Vite messages.  I'm using Golang [Task](https://github.com/go-task/task) to run Vite in parallel with the `go run`.

**Comment by ganigeorgiev** [comment, marked as answer]
You can disable the debug logs with `--debug=0` (you'll still be able to find your API debug logs in the Admin UI > Logs).
